src/utils/app.utils.ts

```diff
@@ -0,0 +1,68 @@
+import { EventName, EventStatus, StatusMessages } from 'src/common/constants';
+import { FormattedData } from 'src/common/types/webhook.event.types';
+
+export function isPayloadValid(payload: any): boolean {
+  return (
+    payload?.action &&
+    payload?.pull_request &&
+    payload?.repository &&
+    payload?.sender &&
+    payload?.installation
+  );
```

Comment:
PrimathonAICodeReviewer

Consider adding more specific type annotations instead of using `any` for the `payload` parameter in the `isPayloadValid` function for better type safety.



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->